components/apimgt/org.wso2.carbon.apimgt.gateway/src/main/java/org/wso2/carbon/apimgt/gateway/utils/GatewayUtils.java

```diff
@@ -735,6 +736,12 @@ public static AuthenticationContext generateAuthenticationContext(String tokenSi
                                 .equals(subscriptionTierObj.getAsString(APIConstants.JwtTokenConstants.QUOTA_TYPE))) {
                     authContext.setIsContentAware(true);;
                 }
+                if (APIConstants.GRAPHQL_API.equals(synCtx.getProperty(APIConstants.API_TYPE))) {
+                    Integer graphQLMaxDepth = (int) (long) subscriptionTierObj.get("graphQLMaxDepth");
+                    Integer graphQLMaxComplexity = (int) (long) subscriptionTierObj.get("graphQLMaxComplexity");
```

Comment:
isn't "graphQLMaxComplexity" and "graphQLMaxDepth" string used in multiple places? if so can you add it to a string constant class and reuse?